src/scroll-timeline-css-parser.js

```diff
@@ -300,24 +316,29 @@ export class StyleParser {
   }
 
   saveRelationInList(rule, timelineNames, animationNames) {
-    if (animationNames.length == 0) {
-      for (let i = 0; i < timelineNames.length; i++) {
-        this.cssRulesWithTimelineName.push({
-          selector: rule.selector,
-          'animation-name': undefined,
-          'animation-timeline': timelineNames[i]
-        });
-      }
-    } else {
-      for (let i = 0; i < Math.max(timelineNames.length, animationNames.length); i++) {
-        this.cssRulesWithTimelineName.push({
-          selector: rule.selector,
-          'animation-name': animationNames[i % animationNames.length],
-          'animation-timeline': timelineNames[i % timelineNames.length]
-        });
-      }
-    }
+    const hasAnimationDelay = rule.block.contents.includes("animation-delay:");
+    const hasAnimationEndDelay = rule.block.contents.includes("animation-end-delay:");
+
+    let animationDelays = [];
+    let animationEndDelays = [];
```

Comment:
Note that endDelay though a property of animation-effect-timing does not map to a CSS longhand property name.  Probably deserving of a comment.  We'll need to update this in the css-animations-2 (?) spec.